snippets/brightscript.json

```diff
@@ -0,0 +1,71 @@
+{
+    "CreateObject": {
+        "prefix": "createObject",
+        "body": [
+            "${2:variableName} = CreateObject(\"${1:objectType}\")"
+        ],
+        "description": "Simple CreateObject"
+    },
+
+    "For Each": {
+        "prefix": "for-each",
+        "body": [
+            "for each ${1:item} in ${2:object}",
+            "\t$0",
+            "end for"
+        ],
+        "description": "For Each item in a collection"
+    },
+
+    "Function": {
+        "prefix": "function",
+        "body": [
+            "function ${1:functionName}()",
```

Comment:
Should we have a placeholder for params?

Reply:
Yes, we should...

Reply:
I updated the PR with params placeholder in functions/subs